### ISSUE TITLE
Automatic insertion of colons after 'for', 'if', 'def' etc is not working 

### ISSUE THREAD

**Issue by irnmn** [spyder-ide/spyder#11921]
USing spyder 4.1.1 
python 3.7
after if statement or def and push enter no automatic semicolumn 

for example `if playerMove == 'r' ` (push enter after this)
does not work. Indentation works though!

![image](https://user-images.githubusercontent.com/13536632/77278123-c29bf780-6c7b-11ea-861a-2d9e88b412ed.png)

**Comment by TARDIInsanity** [comment]
Yes. i agree this is an issue: spyder specifically doesn't (or can't) insert colons after *strings* specifically. for example if you surrounded that condition in redundant parens, it would insert properly, and after doing 100 or so string equality checks it gets kinda annoying to need to do this manually but not for any other colons

**Comment by nicolas-joffre** [comment]
Still an issue in Spyder 6.1.6
